PyZAPI/zappa_settings.json

```diff
@@ -0,0 +1,9 @@
+{
+    "default": {
+        "app_function": "pyzapi.app.zappa_dev",
+        "aws_region": "us-east-1",
+        "profile_name": "default",
+        "s3_bucket": "zappa-0oqiolisl",
```

Comment:
As I mentioned with the developer.ini etc., you don’t want to hard code the s3 bucket name in here, you probably want to create a zappa.json.example maybe? Then have folks add what they need in here?

I think these are good opportunities for using our script system… that being said, I’ll probably just let this pass through and then put a PR in myself with the changes to show you what I’m thinking.